src/types.ts

```diff
@@ -21,7 +21,10 @@ export type Env = {
 
 export type Next = () => Promise<void>
 
-export type Input = unknown
+export type Input = {
```

Comment:
`Input["input"]` and `Input["output"]` might be confusing, so there would be a better name for this (I'm afraid I can't come up with a nice suggestion now). Ideas from other people are very much appreciated.

Reply:
I've renamed `input` to `in` and `output` to `out`. It's not that different, but I think it's easier to understand than before.